orderly_web/start.py

```diff
@@ -115,9 +117,19 @@ def web_container(cfg, docker_client):
     container = docker_client.containers.run(
         image, mounts=mounts, network=cfg.network, ports=ports,
         name=cfg.containers["web"], detach=True)
+    if cfg.logo_name is not None:
+        generate_favicon(cfg.logo_path)
+        file_into_container("favicon.ico", container, "/static/public")
+        os.remove("favicon.ico")
     return container
 
 
+def generate_favicon(logo_path):
+    filename = logo_path
+    img = Image.open(filename)
+    img.save('favicon.ico')
```

Comment:
Can you do this through a temporary file (better in case of things like a conversion failure, etc)